### PR TITLE
#99 - Created text inputs and added its styles

### PR DESCRIPTION
**Pull request by Hanna-N9**

### CODE REVIEW

**Comment by Hanna-N9** [comment]
What it looks like:

4F93FAA8-DEF2-4C61-9888-8787AA98B481_1_105_c

File: client/src/TextInput.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <div>
+        <div className="mt-5 mb-2">
+          <input
+            className={inputStyle}
+            type="text"
+            placeholder="Email address"
+          />
+        </div>
+        <div>
+          <input className={inputStyle} type="text" placeholder="Password" />
```
Comment (mikelbrierly) [inline client/src/TextInput.js:18]: By making this component contain both text inputs, we are actually making it less reusable. If we make this just for one single text input, then we could actually pass in props for the `placeholder`, and it's instantly more valuable as a component.

(Sorry if the acceptance criteria was misleading)

File: client/src/TextInput.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+
+var inputStyle =
+  "bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400";
+
+class TextInput extends Component {
```
Comment (mikelbrierly) [inline client/src/TextInput.js:3]: We might want to consider sticking with a functional component here, unless we have a good reason to switch to a class. This is an interesting SO question that might be helpful - https://stackoverflow.com/questions/36097965/when-to-use-es6-class-based-react-components-vs-functional-es6-react-components

Comment (Hanna-N9) [reply client/src/TextInput.js:3]: Understandable, and it makes sense after reading on StackOverflow about the component.  As they say, technology is changing. Thank you for sharing the link.

File: client/src/TextInput.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      <div className="mt-5 mb-2">
+        <input
+          className="bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400"
+          type="text"
+          placeholder="Email address"
+        />
+      </div>
+      <div>
+        <input
+          className="bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400"
```
Comment (mikelbrierly) [inline client/src/TextInput.js:15]: If we take a look at the two divs you're returning here, they are almost identical except for the placeholder property. In the spirit of keeping our code DRY (don't repeat yourself), how can we refactor this code to return a single input but still be reusable for different placeholders?

File: client/src/TextInput.js [hunk 4]
```diff
@@ -1,22 +1,14 @@
...
-          className="bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400"
-          type="text"
-          placeholder="Password"
-        />
-      </div>
+      <input
+        input
+        className="bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400 mt-2"
+        type="text"
+        placeholder={placeholder}
```
Comment (mikelbrierly) [inline client/src/TextInput.js:9]: Awesome!!! This is perfect

**Review by mikelbrierly — APPROVED** [review]

**Pull request merged**